addon/components/horizontal-bar-chart.js

```diff
@@ -244,19 +246,26 @@ export default ChartComponent.extend(FloatingTooltipMixin,
       },
       y: this.get('barThickness') / 2,
       dy: '.35em',
-      'text-anchor': (d) => d.value < 0 ? 'end' : 'start',
+      'text-anchor': (d) => {
+        if ((d.value < 0) || ((d.value === 0) && this.get('hasNegativeValues') && !this.get('hasPositiveValues'))) {
```

Comment:
Maybe the name needs more thought, but either way, the line 
```
((d.value < 0) || ((d.value === 0) && this.get('hasNegativeValues') && !this.get('hasPositiveValues')))
```
is in this PR four times, so it should probably be abstracted. 
```